cmd/root.go

```diff
@@ -98,6 +99,7 @@ func NewCmdRoot() *cobra.Command {
 	rootCmd.AddCommand(NewCmdCompletion())
 	rootCmd.AddCommand(NewHelpCmd(f))
 	rootCmd.AddCommand(NewOpenCmd(f))
+	rootCmd.AddCommand(filescmd.NewFilesCmd(f))
```

Comment:
I thought the feature request was

```
	sdpctl appliance files ...
```
not 
```
sdpctl files ...
```

Reply:
moved it now